metals/src/main/scala/scala/meta/internal/metals/Decompiler.scala

```diff
@@ -0,0 +1,41 @@
+package scala.meta.internal.metals
+
+import java.nio.file.Paths
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+import scala.meta.internal.metals.MetalsEnrichments._
+
+import com.google.gson.JsonPrimitive
+import org.eclipse.{lsp4j => l}
+import java.{util => ju}
+
+class Decompiler(
+    compilers: Compilers
+) {
```

Comment:
Do we need a separate class for this? We could include this in `Compilers`

Reply:
I find separate file more readable because`Compilers` already have ~700loc. Also, separate file plays nicely with single responsibility - tasty stuff is extracted to a tasty file. Any chance that I convinced you? ;)